Readme.md

```diff
@@ -9,9 +9,95 @@
+
+Exceptions:
+
+- Tracee supports loading a pre-compiled eBPF file, in which case the kernel headers are required only for the one-time compilation, and not at runtime. See Setup Options for more info.
+- When using Tracee's Docker image, all of the aforementioned requirements are built into the image. The only requirement left is the kernel headers or the pre-built eBPF. See Setup Options for more info.
+
+### Quickstart with Docker
+
+```bash
+docker run --name tracee --rm --privileged -v /lib/modules/:/lib/modules/:ro -v /usr/src:/usr/src:ro -v /tmp/tracee:/tmp/tracee aquasec/tracee:latest
+```
+
+> Note: You may need to change the volume mounts for the kernel headers based on your setup. See [Linux Headers](#Linux-Headers) section for info.
+
+This will run Tracee with no arguments, which defaults to detecting all available rules and reporting them as raw messages on standard output of the container. These can be further customized as detailed below.
+
+### Rules
+
+To view the list of available rules, run the container with the `--list` flag.
+
+To select a specific rules, use the `--rules` flag which accepts a comma separated list of rules names.
+
+### Integrations
+
+Tracee leverages [falco-sidekick](https://github.com/falcosecurity/falcosidekick) for sending it's detection events into other systems which are easier to consume. You can use any of falco-sidekick's supported "outputs", which includes: Slack, Mattermost, Teams, Datadog, Prometheus, StatsD, Email, Elasticsearch, Loki, PagerDuty, OpsGenie, and many more. The full list is available [here](https://github.com/falcosecurity/falcosidekick#outputs).
+
+To configure Tracee to integrate with another system, compose a falco-sidekick configuration file, and provide it to Tracee using the `TRACEE_WEBHOOK_CONFIG` environment variable. By default, Tracee will try to find this file at `/tmp/tracee/integrations-config.yaml`, so if you have followed the quickstart and started the container with `/tmp/tracee` mounted in, you can simply drop that file there.
+
+A complete reference of falco-sidekick's configuration format is available [here](https://github.com/falcosecurity/falcosidekick/blob/master/config_example.yaml).
+
+### Setup options
+
+Tracee is leveraging Linux's eBPF technology, which is kernel and version sensitive, and therefore needs to be specifically compiled for your hosts.
+
+The easiest way to get started is to just let Tracee build the eBPF program for you automatically when it starts, as demonstrated by the [Quickstart](#quickstart).  
+Alternatively, you can pre-compile the eBPF program, and provide it to Tracee. There are some benefits to this approach as you will not need clang and kernel headers at runtime anymore, as well as reduced risk of invoking an external program at runtime.
+
+You can build the eBPF program in the following ways:
+1. Clone the repo and `make bpf`.
+2. `make bpf DOCKER=1` to build in a Docker container which includes all development tooling.
```

Comment:
I'll add this in a future PR. don't want to make this PR larger then it already is

Reply:
#519